tests/test_caching.py

```diff
@@ -831,7 +831,7 @@ def test_solve(self, operate_on_empty_cache):
         # created by the finite difference (u.dt, u.dx2). We would have had
         # three extra references to u(t + dt), u(x - h_x) and u(x + h_x).
         # But this is not the case anymore!
-        assert len(_SymbolCache) == 11
+        assert len(_SymbolCache) == 12
```

Comment:
Note: This is because now every order uses a StencilDImension even for `space_order=1` while we were only giving the list of indices for that case before.